.github/actions/run-server/action.yml

```diff
@@ -7,8 +7,7 @@ inputs:
     default: ci
   database_url:
     description: DATABASE_URL to set
-    required: false
-    default: postgres://pangolin:postgres@localhost:5432/pangolin_test
+    required: true
```

Comment:
In the spirit of only having to update the url string in one place, requiring this to be passed in from the setup action outputs.

Reply:
🎉